mne/viz/_3d.py

```diff
@@ -1041,39 +1143,70 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
                 will be mirrored directly across zero during colormap
                 construction to obtain negative control points.
 
+        Has no effect with mpl backend.
     cortex : str or tuple
-        specifies how binarized curvature values are rendered.
-        either the name of a preset PySurfer cortex colorscheme (one of
-        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the
-        name of mayavi colormap, or a tuple with values (colormap, min,
-        max, reverse) to fully specify the curvature colors.
+        Specifies how binarized curvature values are rendered.
+        Either the name of a preset PySurfer cortex colorscheme (one of
+        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the name of
+        mayavi colormap, or a tuple with values (colormap, min, max, reverse)
+        to fully specify the curvature colors. Has no effect with mpl backend.
     size : float or pair of floats
         The size of the window, in pixels. can be one number to specify
         a square window, or the (width, height) of a rectangular window.
+        Has no effect with mpl backend.
     background : matplotlib color
-        Color of the background of the display window.
+        Color of the background of the display window. Has no effect with mpl
+        backend.
     foreground : matplotlib color
-        Color of the foreground of the display window.
+        Color of the foreground of the display window. Has no effect with mpl
+        backend.
     initial_time : float | None
         The time to display on the plot initially. ``None`` to display the
         first time sample (default).
     time_unit : 's' | 'ms'
         Whether time is represented in seconds ("s", default) or
         milliseconds ("ms").
-
+    backend : 'auto' | 'mayavi' | 'mpl'
+        Which backend to use. If ``'auto'`` (default), tries to plot with
+        mayavi, but resorts to matplotlib if mayavi is not available.
 
     Returns
     -------
     brain : Brain
-        A instance of surfer.viz.Brain from PySurfer.
-    """
-    import surfer
-    from surfer import Brain, TimeViewer
-    import mayavi
-
+        A instance of surfer.viz.Brain from PySurfer or matplotlib figure.
+    """  # noqa: E501
     # import here to avoid circular import problem
     from ..source_estimate import SourceEstimate
-
+    if not isinstance(stc, SourceEstimate):
+        raise ValueError('stc has to be a surface source estimate')
+    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                    raise_error=True)
+    subject = _check_subject(stc.subject, subject, True)
+    if backend not in ['auto', 'mpl', 'mayavi']:
+        raise ValueError("backend must be 'auto', 'mayavi' or 'mpl. "
```

Comment:
'mpl -> 'matplotlib'

I prefer the full name matplotlib

mpl is not standard.